rust/numaflow-grpc/src/clients.rs

```diff
@@ -0,0 +1,31 @@
+#[path = "clients/sourcetransformer.v1.rs"]
+#[rustfmt::skip]
```

Comment:
I think we should add  ignore annotation for `clippy` too.

Reply:
`#[allow(clippy::all)]` - Is this right? @BulkBeing

Reply:
perhaps `[#![warn(clippy::style)]` ?

Reply:
I think it should be `#[allow(clippy::all)]`. The `#![warn(clippy::style)]` is enabling style warnings for the whole crate. I tried running clippy on `numaflow-grpc`, there are no warnings. The openapi generated code has a lot of warnings.

Reply:
The generated code already has the necessary annotations. So we don't have to do anything.
```rust
/// Generated client implementations.
pub mod map_stream_client {
    #![allow(
        unused_variables,
        dead_code,
        missing_docs,
        clippy::wildcard_imports,
        clippy::let_unit_value,
    )]
    use tonic::codegen::*;
```